src/keri/app/cli/kli.py

```diff
@@ -25,7 +25,12 @@ def main():
         directing.runController(doers=doers, expire=0.0)
 
     except Exception as ex:
-        print(f"ERR: {ex}")
+        import os
+        if os.getenv('DEBUG_KLI'):
+            import traceback
+            traceback.print_exc()
+        else:
+            print(f"ERR: {ex}")
         return -1
         # raise ex
```

Comment:
Minor nit... can we remove this commented line since its no longer needed!

Reply:
Done (commit 757b7cb)